src/helper/sqlstring.js

```diff
@@ -0,0 +1,55 @@
+const knexClientFactory = require('knex');
+
+function format(sql, bindings, dialect) {
+  const knexClient = knexClientFactory({ client: dialect });
+
+  const stringBindings = bindings
+    ? bindings.map((binding) => {
+        if (!binding) {
+          return binding;
+        }
```

Comment:
This will check for all `falsy` values. Is it intended?

Reply:
Actual purpose of the method is to set the brinding for types of (Date, Infinity, Array or Object) and return the same for all the others. Copied this from a library, thinking back can't think of a reason why they'd add this check though. It anyway returns binding at the end. But since the library works well and has no unit tests, I'll set a task to look at this one again after adding unit tests.